packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -149,7 +167,7 @@ export const Settings = ({
       <Head />
       <Router basepath={SETTINGS_PATH}>
         <ScrollToTop default>
-          <PageSettings path="/" />
+          <PageSettings path="/" isInactiveVerified={showInactiveBanner()} />
```

Comment:
This bit doesn't have the same ergonomics as the rest of the code in this component. I think it needs improvement; open to ideas.